conference.rb

```diff
@@ -0,0 +1,31 @@
+require_relative "talks" 
+require_relative "session"
+require_relative "track"
+
+class Conference
+  attr_reader :tracks, :session
+
+  def initialize(session)
+    @tracks = []
+    @session = session
```

Comment:
Why does the conference have a reference to session? A conference has only tracks, not sessions. A track has sessions and hence the session should be used within a track only.